src/coreclr/vm/ceeload.cpp

```diff
@@ -489,13 +489,11 @@ void Module::Initialize(AllocMemTracker *pamTracker, LPCWSTR szName)
         m_dwTransientFlags |= PROF_DISABLE_OPTIMIZATIONS;
     }
 
-#if !defined(DACCESS_COMPILE)
```

Comment:
How is this related to the other code removal?

Reply:
This is unrelated to the UnhandledException dead code removal. (I had this edit in my deadcode branch where I am accumulating small dead code deletes when I run into them.)

Reply:
There is identical ifdef around this whole method.